ckanext/geocat/utils/csw_mapping.py

```diff
@@ -128,11 +146,19 @@ def get_metadata(self, csw_record_as_string, geocat_id):
                                    organization_slug=self.organization_slug,
                                    valid_identifiers=self.valid_identifiers)
         dataset_dict['owner_org'] = self.organization_slug
+
         rights = \
             _map_dataset_rights(node=root_node,
```

Comment:
In this method, we get the rights for the dataset, but only if they belong to the four opendata.swiss terms of use (see line 446). We need to remove this restriction if we are going to get any other licences/terms of use.

Reply:
I asked, if I should add the cc licenses for the geocat terms of use and as I understand we decided in the daily, that we leave it like this, because otherwise there could be the case, that there is only a cc license.
So if there is now cc license is coming in, we will use the default term that was configured in the harvester (see also last comment in the ticket)?

Reply:
Oh yes, you're right, I had forgotten about that. :) That does make things simpler here.